src/boot/libc64/__osMalloc.c

```diff
@@ -22,7 +23,7 @@ OSMesg sArenaLockMsg[1];
 void __osMallocAddHeap(Arena* arena, void* heap, size_t size);
 
 void ArenaImpl_LockInit(Arena* arena) {
-    osCreateMesgQueue(&arena->lock, sArenaLockMsg, ARRAY_COUNT(sArenaLockMsg));
+    osCreateMesgQueue(&arena->lock, sArenaLockMsg, sizeof(sArenaLockMsg) / sizeof(*sArenaLockMsg));
 }
```

Comment:
Instead of removing the macro here, I wonder if it would be better to have a more general "macros.h" that would include common macros that libraries/the game would share (e.g. ultalib has a macros.h). We could rename the other one to something like `z64macros` that contains game specific macros.

Reply:
On an ideal world `macros.h` would only contain generic macros and each specific macro would be placed on the corresponding header. For example, `GET_PLAYER` would be in `z64play.h`, `MAGIC_FILL_TO_CAPACITY` would be in either `z64save.h` or `z64interface.h`, etc.

I guess it may not be worth the trouble waiting, and we should just use `macros.h` on the library files

Reply:
The main thing is I don't want us to remove the macros, and than eventually add them (especially since we might miss some). Seems better to just either split macros.h or just use it straight up for the time being. (tbh ARRAY_COUNT is the one I really don't want to be removed, `SQ` isn't as important to me.)